content/docs/command-reference/commit.md

```diff
@@ -70,6 +70,8 @@ force-update the [DVC-files](/doc/user-guide/dvc-file-format) and save data to
 cache. They are still useful, but keep in mind that DVC can't guarantee
 reproducibility in those cases.
 
+Note that [patterns](https://git-scm.com/docs/gitignore) listed in `.dvcignore` are not updated as a result of `dvc commit` as they are not currently tracked by DVC. See [.dvcignore](docs/user-guide/.dvcignore) for more details.
+
```

Comment:
This note was in a more appropriate place. The paragraph just needed formatting (see docs contrib guide) and also I would not link `patterns` to the Git website in any of these docs to avoid confusions.

Reply:
The link href is incorrect though, it's missing a `/` in the beginning, please run the app locally so you can see and try your changes. See docs contrib guide.